### ISSUE TITLE
[Story] Completely update all base-python connectors, use CDK package instead

### ISSUE THREAD

**Issue by yevhenii-ldv** [airbytehq/airbyte#7242]
Part of #6164 epic

Currently we have a lot of connectors using `base-python` package.
In this story, we need to use the classes and methods from the `СDK` instead of using them from the `base-python` package for following connectors:

**Note**: Please, before starting the new task - please create a separate issue for it and edit the description by adding a link.

- [x] source-s3 
- [x] source-apify-dataset
- [x] source-bigcommerce 
- [x] source-dixa 
- [x] source-drift #7041
- [x] source-exchange-rates
- [x] source-file-secure #7447
- [x] source-file #7336 7336
- [x] source-freshdesk #7764
- [x] source-google-directory #7415 
- [x] google-sheets-source #8313 
- [x] source-google-workspace-admin-reports  #7531 
- [x] source-hubspot #7521
- [x] source-intercom
- [x] ~~source-jira~~ - deprecated dependency was removed in #7408
- [x] source-linkedin-ads
- [x] source-looker #7246
- [x] source-microsoft-teams #8315
- [x] source-mixpanel
- [x] source-paypal-transaction
- [x] source-pokeapi #8422
- [x] source-python-http-tutorial (low priority IMO!) #8423
- [x] source-recurly #8318
- [x] source-scaffold-source-http
- [x] source-scaffold-source-python
- [x] source-shortio
- [x] source-slack
- [x] source-smartsheets
- [x] source-snapchat-marketing
- [x] source-square
- [x] source-tiktok-marketing
- [x] source-trello
- [x] source-us-census
- [x] source-woocommerce
- [x] source-zendesk-chat #8421
- [x] source-zendesk-talk #6599

**Comment by htrueman** [comment]
These connectors were refactored in https://github.com/airbytehq/airbyte/pull/7499, so no need to create a separate issues for them:
`source-s3`,
`source-apify-dataset`,
`source-bigcommerce`,
`source-dixa`,
`source-apify-dataset`,
`source-bigcommerce`,
`source-dixa`,
`source-exchange-rates`,
`source-intercom`,
`source-linkedin-ads`,
`source-mixpanel`,
`source-paypal-transaction`,
`source-s3`,
`source-scaffold-source-http`,
`source-scaffold-source-python`,
`source-shortio`,
`source-slack`,
`source-smartsheets`,
`source-snapchat-marketing`,
`source-square`,
`source-tiktok-marketing`,
`source-trello`,
`source-us-census`,
`source-woocommerce`
